common.sh

```diff
@@ -2,6 +2,7 @@
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 USER=`whoami`
+WORKING_DIR="/opt/dev-scripts"
```

Comment:
We should make this overrideable e.g WORKING_DIR=${WORKING_DIR:-"/opt/dev-scripts"} - that means there's a workaround if anyone wants to temporarily use the /home/stack location by setting it in the per-user config files.

Reply:
yep, I will change that